packages/hermes/src/email/mailer.ts

```diff
@@ -0,0 +1,18 @@
+import nodemailer from "nodemailer";
+const mg = require("nodemailer-mailgun-transport");
```

Comment:
Why use `require` here?

Reply:
Good point. Was a lazy copy and paste from the package docs.